project/Dependencies.scala

```diff
@@ -48,7 +48,7 @@ object Dependencies {
   lazy val test = Seq(
     "junit" % "junit" % "4.13.2" % "it,test",
     "com.novocode" % "junit-interface" % "0.11" % "it,test",
-    "org.scalacheck" %% "scalacheck" % "1.15.4" % "it,test"
+    "org.scalacheck" %% "scalacheck" % "1.15.4" % "it,test",
   )
```

Comment:
I don't mind adding a trailing comma to the last item in a Seq (Scala explicitly [allows](https://docs.scala-lang.org/sips/trailing-commas.html) it for good reasons), but the previous Seq's don't use a trailing comma so we've introduced an inconsistency.  Maybe fix all the other Seqs too?